safe-global/web-core
# feat: default token list toggle

## What it solves
- Adds default token list toggle
- Improves assets loading state

Resolves #865 

## Todo

## How this PR fixes it
tbd

## How to test it
On staging CGW and görli / mainnet:
- Go to Assets table
- Observe new Select for tokenlist
- By default its "Default list"
- when switching to "All" the balances are reloaded and will show all tokens.

## Analytics changes
tbd

## Screenshots
tbd
